tests/features/data-visibility-and-review.feature

```diff
@@ -1,122 +1,202 @@
-#  @skip
-#  @authorization
-#  Scenario: Allow all datasets for internal users
-#    Given the system has datasets with combinations of:
-#      | visibility | review_status |
-#      | internal   | provisional   |
-#      | internal   | approved      |
-#      | public     | provisional   |
-#      | public     | approved      |
-#    And the caller is identified as an internal staff user
-#    When the internal staff user requests those datasets via the api
-#    Then the system should return all of those datasets in the response
-#
-#  @skip @authorization
-#  Scenario: Reject unauthorized changes to visibility or review status
-#    Given the system has a dataset with visibility "internal" and review_status "provisional"
-#    And the caller is authenticated without permission to change visibility or review_status
-#    When the caller attempts to update the visibility to "public" via the api
-#    And the caller attempts to update the review_status to "approved" via the api
-#    Then the system should return an authorization error for these updates
-#    And the dataset should remain persisted with visibility "internal" and review_status "provisional"
-#
-#  @skip
-#  @disclaimer
-#  Scenario: Include disclaimer information based on review status
-#    Given the system has a dataset with visibility "public" and review_status "provisional"
-#    When any authorized caller retrieves the dataset via the api
-#    Then the system should return a response in JSON format
-#    And the response should include a disclaimer field derived from the review_status
-#    And the disclaimer should indicate that the data is provisional
-#
-#    And the system has a dataset with visibility "internal" and review_status "approved"
-#    When any authorized caller retrieves that dataset via the api
-#    Then the response should include a disclaimer field derived from the review_status
-#    And the disclaimer should indicate that the data is approved
-#
+# Created by marissa at 12/11/2025
+# file: tests/features/data_visibility_and_review_backend.feature
+
+@backend @BDMS-XXX
+Feature: Manage data visibility separately from review in the backend
+  As an Ocotillo data manager
+  I want visibility and review to be stored as two separate backend attributes
+  So that the system can independently control who can see the data and which data is reviewed
+
+  # Business rules:
+  # - visibility has two states: "internal" or "public"
+  # - review_status has two states: "provisional" or "approved"
+  # - visibility and review_status are REQUIRED when creating new data
+  # - new data must use visibility and review_status (no new use of legacy fields)
+  # - legacy combined values will be migrated into visibility and review_status using a documented mapping
+  # - visibility and review must cover previous release_status business logic
+
+  Background:
+    Given a functioning api
+    And all database models have a visibility field that supports internal and public
```

Comment:
I think that `supports internal and public` is somewhat vague. What about the following?

> And all database models have a visibility field that determines if a record can or cannot be viewed by the public

or

> And all database models have a visibility field that if a record is for internal purposes only, or if it can always be viewed by the public

Reply:
If `internal` and `public` are the restricted vocabulary, I think that should be another step:

`And the only possible values for the visibility field are "internal" and "public"`